src/scss/page-types/_campaign-page.scss

```diff
@@ -68,6 +68,12 @@
       align-items: center;
     }
 
+    .su-lockup__logo-cell1 {
+      max-width: 160px;
+      max-height: 100px;
```

Comment:
I added some minimal guards so they can't terribly destroy the existing layout 🤷🏼